docs/spec_draft.md

```diff
@@ -420,6 +421,47 @@ IEEE-754 specification.
 
 [Back to Ops](#index-of-documented-ops)
 
+## stablehlo.log
+
+`stablehlo.log(operand) -> result`
+
+### Semantics
+
+Performs element-wise logarithm operation on `operand` tensor and
+produces a `result` tensor, implementing the `log` operation from the IEEE-754
```

Comment:
As per the IEEE standard `log` is not meant for the complex op, can we re-frame as

`Performs element-wise logarithm operation on `operand` tensor and produces a `result` tensor. For floating-point element types, implements the `log` from ...`.

Reply:
Also same comment for the already merged ops like sqrt,rsqrt,sine, cos which can be handled via a separate PR.

Reply:
How does this apply to other ops which work on complex numbers, like sine and cosine?

Reply:
Say for sine, we have the following spec 

```
Performs element-wise sine operation on `operand` tensor and produces a `result`
tensor, implementing the `sin` operation from the IEEE-754
specification. Numeric precision is implementation-defined.
```

Again `sin` operation from IEEE is not meant for complex type.

Reply:
If we don't explicitly state the behavior of complex numbers, the element-wise x operation can be misinterpreted as performing x operation on the real and imaginary values separately when in reality it is computed together. Since complex numbers are not defined in IEEE-754, should the behavior be explicitly stated similar to that of [C++ documentation](https://en.cppreference.com/w/cpp/numeric/complex/log)? @burmako

Reply:
Huh that's a good catch. 

How about deferring describing corner cases for complex numbers to a later point? It feels that this will take a significant chunk of time while not being essential for the first pass over the spec. We could say something along these lines:

"""
Performs element-wise logarithm operation on `operand` tensor and produces a `result` tensor. For floating-point element types, implements the `log` operation from the IEEE-754 specification. For complex element types, computes a complex logarithm, with corner cases TBD. Numeric precision is implementation-defined.
"""